compound-comptroller-bot/src/agent.spec.ts

```diff
@@ -0,0 +1,171 @@
+import { Interface } from "ethers/lib/utils";
+import {
+  FindingType,
+  FindingSeverity,
+  Finding,
+  HandleTransaction,
+  TransactionEvent,
+  ethers,
+  Network,
+} from "forta-agent";
+import { NetworkManager } from "forta-agent-tools";
+import { TestTransactionEvent } from "forta-agent-tools/lib/test";
+import { createAddress } from "forta-agent-tools/lib";
+import { provideHandleTransaction } from "./agent";
+import { AgentConfig, NetworkData, PAUSE_EVENTS_ABIS } from "./utils";
+
+const COMPOUND_COMPTROLLER_ADDRESS = createAddress("0xcc01");
+
+const DEFAULT_CONFIG: AgentConfig = {
+  [Network.MAINNET]: {
+    compoundComptrollerAddress: COMPOUND_COMPTROLLER_ADDRESS,
+  },
+};
+
+const createFinding = (signature: string, args: any[]): Finding => {
```

Comment:
`Finding`s here will also need to have the proper value to the `addresses` property.